bazel/external_crates.bzl

```diff
@@ -567,6 +569,12 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
                     "pem",
                 ],
             ),
+            "ic-agent-call-v3": crate.spec(
```

Comment:
Please add a todo with. a jira ticket attached that we should delete this once the feature is done